src/cryptonote_basic/tx_extra.h

```diff
@@ -39,6 +39,8 @@
 #define TX_EXTRA_NONCE                      0x02
 #define TX_EXTRA_MERGE_MINING_TAG           0x03
 #define TX_EXTRA_GRAFT_EXTRA_TAG            0x04
+#define TX_EXTRA_GRAFT_STAKE_TX_TAG         0x05
```

Comment:
It make sense to reserve some gap after monero constants (TX_EXTRA_GRAFT_EXTRA_TAG will be removed most lilely as it only used by old supernode, was selected without and assumptions about future compatibility with monero)
in monero 0x04 already used